testsuite/tests/kuadrant/authorino/dinosaur/conftest.py

```diff
@@ -282,3 +288,57 @@ def user_with_write_role(admin_rhsso, blame):
     user = admin_rhsso.realm.create_user(blame("someuser"), blame("password"))
     user.assign_realm_role(admin_rhsso.realm.create_realm_role("admin-write"))
     return HttpxOidcClientAuth.from_user(admin_rhsso.get_token, user=user)
+
+
+@pytest.fixture(scope="module")
+def delete_jwks_key(rhsso):
```

Comment:
I think these don't need to be fixtures, they can be standard functions

Reply:
I did not get what you meant by this but after Phil's comment below I started to think that you meant exactly what he suggested. I force pushed the turn of these functions to RHSSO methods.